test/test_common/BUILD

```diff
@@ -49,6 +49,17 @@ envoy_cc_test_library(
     ],
 )
 
+envoy_cc_test_library(
+    name = "contention_lib",
```

Comment:
I had it as part of the `mutex_tracer_test.cc`, and extracted it for use in `main_common_test.cc` (AdminRequestTest.AdminRequestContentionEnabled), since I wanted to induce contention there too.